Tick2q/Program.fs

```diff
@@ -9,6 +9,8 @@ open FSharp.Core
 open Simulate
 open Simulate.MapEnvt
 
+/// The test runs a simulated 4-bit adder for 17 steps. Your task is to implement the functions in simulate.fs and ensure
+/// the correct function of this test: The output of the adder should be what you expect over 17 cycles, given the inputs.
```

Comment:
I was quite lost when I first opened the file. Granted, this was without the context of the video that students would have watched. Nonetheless, this should make it clear.